metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -50,7 +50,7 @@ import (
 )
 
 var metricSets = []string{
-	"ccr",
+	//"ccr",
```

Comment:
@ycombinator @cachedout The build goes green after disabling ccr here. Are you ok with merging it like this? 

I'm wondering if it's the same as here: https://github.com/elastic/beats/issues/10948 Today I saw quite a few PR's that failed like this.

Reply:
One more thing: I only managed to reproduce this locally when using the master build and not 6.6. I wonder if this is tested against 6.6 or master now :-(

Reply:
I moved to use php_fpm instead to not have to deal with it.